spec/rails_helper.rb

```diff
@@ -21,6 +22,9 @@
   config.allow_url 'cdn.ravenjs.com'
 end
 
+Capybara::Screenshot.webkit_options = { width: 1400, height: 768 }
+Capybara.save_and_open_page_path = "tmp/screenshots/"
```

Comment:
We should consider having a cron task that clears this directory out every so often so it doesn't get bloated.